conf/routes

```diff
@@ -21,7 +21,7 @@ POST    /api2/atoms/:id/assets          controllers.Api2.addAsset(id)
 
 PUT     /api2/atom/:id/publish          controllers.Api2.publishMediaAtom(id)
 PUT     /api/atom/:id/update-metadata   controllers.Api2.updateMetadata(id)
-POST    /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
+PUT     /api2/atom/:id/asset-active     controllers.Api2.setActiveAsset(id)
```

Comment:
This'll require a change to CDS too - https://github.com/guardian/content_delivery_system/blob/c96aa063527edef19badbc060b317c20705dadc3/CDS/scripts/js_utils/media-atom-lib.js#L127

We'd need to be careful on the deploy order.